sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/DiscountTagService.java

```diff
@@ -0,0 +1,7 @@
+package com.exadel.sandbox.team5.service;
+
+import com.exadel.sandbox.team5.entity.DiscountTag;
+
+public interface DiscountTagService extends CRUDService<DiscountTag>{
```

Comment:
It is redundant service. You should get discount's tags using `DiscountService` because tags are part of Discount in this case